etc/config/spice.amazon.properties

```diff
@@ -0,0 +1,40 @@
+compilers=&spice
+defaultCompiler=spice01902
+
+group.spice.compilers=spice01902
+group.spice.demangler=/opt/compiler-explorer/gcc-12.1.0/bin/c++filt
+group.spice.objdumper=/opt/compiler-explorer/gcc-12.1.0/bin/objdump
+group.spice.isSemVer=true
+group.spice.baseName=spice
+group.spice.compilerType=spice
+group.spice.versionFlag=--version
+group.spice.licenseLink=https://github.com/spicelang/spice/blob/main/LICENSE
+group.spice.licenseName=The MIT License
+group.spice.licensePreamble=Copyright (c) 2021-2024 Marc Auberer
+
+compiler.spice01902.exe=/opt/compiler-explorer/spice-0.19.2/spice
+compiler.spice01902.semver=0.19.2
+compiler.spice01902.name=spice 0.19.2
```

Comment:
you don't need the .name field. It should be correctly compited using semVer + baseName.